LeaderboardBackend/Models/Features.cs

```diff
@@ -0,0 +1,7 @@
+namespace LeaderboardBackend.Models;
```

Comment:
Is Models really the right directory for this?
I would default to the project's root folder